### ISSUE TITLE
Webhook: admissionregistration.k8s.io/v1beta1 removed in Kubernetes 1.22++

### ISSUE THREAD

**Issue by robert-heinzmann-logmein** [anchore/anchore-charts#188]
As admissionregistration.k8s.io/v1beta1 is removed in 1.22, helm chart installation failes:

```
shell$ helm install democtl  -f git/anchore-charts/stable/anchore-admission-controller/my_values.yaml git/anchore-charts/stable/anchore-admission-controller/
Error: INSTALLATION FAILED: unable to build kubernetes objects from release manifest: unable to recognize "": no matches for kind "ValidatingWebhookConfiguration" in version "admissionregistration.k8s.io/v1beta1"
```

**Comment by SandPhoenixX517** [comment]
Hello,
I am having the same issue... 
Did you find a solution for this?
Thanks,

**Comment by zhill** [comment]
This is fixed via #228
